jwst/photom/photom.py

```diff
@@ -712,7 +719,13 @@ def photom_io(self, tabdata, order=None):
                 self.input.var_rnoise *= conversion**2
             if self.input.var_flat is not None and np.size(self.input.var_flat) > 0:
                 self.input.var_flat *= conversion**2
-            self.input.dq[no_cal] = np.bitwise_or(self.input.dq[no_cal], dqflags.pixel['DO_NOT_USE'])
+            if no_cal is not None:
+                if len(self.input.dq.shape) == 3:
+                    self.input.dq[:,no_cal] = np.bitwise_or(self.input.dq[:,no_cal],
+                        dqflags.pixel['DO_NOT_USE'])
```

Comment:
This doesn't bother me, but if `flake8` complains, you could move the first argument down to align with this argument.